src/main/kotlin/org/elaastic/questions/assignment/sequence/SequenceService.kt

```diff
@@ -79,6 +81,24 @@ class SequenceService(
         } ?: throw EntityNotFoundException("There is no sequence for id \"$id\"")
     }
 
+    // TODO: delete this method!
+    @Deprecated("Will be deleted",
+                ReplaceWith("findPreviousSequence() and findNextSequence() methods"))
+    fun findSequenceByRankAndAssignment(rank: Int, assignment: Assignment): Sequence
+        = sequenceRepository.findSequenceByRankAndAssignment(rank, assignment)
+            ?: throw EntityNotFoundException(
+                "There is no sequence for id \"$rank\" with assignment id \"${assignment.rank}\"")
```

Comment:
If the method is never used, I propose to remove it.

Reply:
Fix f2a0cc18815cf9746ac49ca69c8cad9c6640b31e